### PR TITLE
[14.0][ADD] partner_bank_code

### PR DESCRIPTION
**Pull request by Saran440**
Add fields Bank Code and Bank Branch Code on Banks.

My country still use `Bank Code` and `Bank Branch Code` for domestic.
I'm not sure other country still used this.

### CODE REVIEW

File: partner_bank_code/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_bank_code/models/__init__.py]: Make license lines https: everywhere...

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Comment by Saran440** [comment]
@NL66278 Fixed, Thank you for your review.

**Review by ps-tubtim — APPROVED** [review]

**Pull request merged**